documentation/updates.md

```diff
@@ -7,34 +7,48 @@ redirect_from:
 ---
 # Updates
 
-## Step 1
+## Step 1 - **Create a full backup!** 
 
-**Create a full backup!** 
+You need to backup both: your Kimai **database** and the **folder containing your Kimai installation**, especially the file `includes/autoconf.php`. To backup the folder holding your Kimai installation you can rename it and create a new folder for the new version. For thes instructions we will use the folders `public_html` and `public_html-backup`. If your webhosting use a different name adjust the instructions below as needed.
 
-You need to backup both: your **database** and your file system, especially the file **includes/autoconf.php**.
+## Step 2 - Download and unzip
+Change to the folder were you will be installing the new version of Kimai and download the newest version with something like:
+```
+wget https://github.com/kimai/kimai/releases/download/1.3.1/kimai_1.3.1.zip
+```
+The link to the newest version can be found on the [Downloads](/download/) page.
 
-## Step 2
+Unzip it with something like this:
```

Comment:
~~something like this~~

Reply:
I added a clarification statement as the reason I phrased it this way is the command run will change with each new version of Kimai. Feel free to improve on this.